discord/message.py

```diff
@@ -376,6 +381,7 @@ def __init__(self, *, state, channel, data):
         self.tts = data['tts']
         self.content = data['content']
         self.nonce = data.get('nonce')
+        self.stickers = [Sticker(state, data) for data in data.get("stickers", [])]
```

Comment:
```suggestion
        self.stickers = [Sticker(data=data, state=state) for data in data.get('stickers', [])]
```

Probably worth investigating the message edit behaviour with stickers and see if it needs an edit handler.